keyboards/vortex/m0110a/info.json

```diff
@@ -340,7 +340,7 @@
                     { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
                     { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
                     { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
-                    { "label": "\\", "matrix": [10, 5], "w": 1.5, "x": 13.5, "y": 1 },
+                    { "label": "\\", "matrix": [10, 2], "w": 1.5, "x": 13.5, "y": 1 },
```

Comment:
Is this accurate? It was initially [10,5] for the ANSI Backslash.

Reply:
@noroadsleft yes, we want to use these two matrix for backslash. It is ok to keep [10,2]. because we want to use [10,2] for split backspace.